inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -1281,6 +1281,23 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node, format
             impl_candidate = impl_types::ocl;
         }
 
+        // [WA] to resolve onednn kernel issue of multiple sum post-ops
+        if (!node.get_fused_primitives().empty()) {
+            size_t eltw_dep = 0;
```

Comment:
what about renaming it `eltw_dep` -> `sum_post_op_cnt` so that it is more clear?

Reply:
Modified